### PR TITLE
added 1 RU string

### PR DESCRIPTION
**Pull request by Tachii**

### CODE REVIEW

**Comment by Tachii** [comment]
Исправил

**Comment by ruPaladin** [comment]
Криво воспринял "<" br ">" поправь на символы аналогичные у других и будет отлично.

File: addons/ballistics/stringtable.xml [hunk 1]
```diff
@@ -1157,7 +1157,8 @@
             <French>Calibre: 6.5x47mm Creedmor &lt;br /&gt;Cartouches: 30&lt;br /&gt;Utilisé avec: MXM</French>
             <Italian>Calibro: 6.5mm Creedmor&lt;br /&gt;Munizioni: 30&lt;br /&gt;In uso su: MXM</Italian>
             <Spanish>Calibre: 6.5mm Creedmor&lt;br /&gt;Balas: 30&lt;br /&gt;Se usa en: MXM</Spanish>
-        </Key>
+            <Russian>Калибр: 6,5x47мм Creedmor&lt;br /&gt;Патронов: 30&lt;br /&gt;Используются c: MXM</Russian>
+	</Key>
```
Comment (KoffeinFlummi) [inline addons/ballistics/stringtable.xml]: Please fix the indentation here (missing space), then I'll merge.

Comment (Tachii) [inline addons/ballistics/stringtable.xml]: done

Comment (Tachii) [inline addons/ballistics/stringtable.xml]: When i'm trying to edit, it shows that everything is as supposed to be, can't find a missing space anywhere on that line...

Comment (commy2) [inline addons/ballistics/stringtable.xml]: It's a tab, but it has to be 8 spaces.

Comment (Tachii) [inline addons/ballistics/stringtable.xml]: well, replaced with alot of spaces instead, should be ok now i guess

**Comment by KoffeinFlummi** [comment]
Awesome, thanks!

**Pull request merged**